### DISCUSSION TITLE
Lost mouse pointer and it's pathetically slow

### DISCUSSION THREAD

**Discussion by sangharshdongre** [prasanthrangan/hyprdots#1740] [Q&A]
I have just installed it on my VM. I don't see mouse pointer and it doesn't respond to the key bindings. I can't to get the terminal open with super + T. Something's terribly wrong.

**Comment by kRHYME7** [comment]
https://wiki.hyprland.org/Getting-Started/Master-Tutorial/#vm

**Reply by sangharshdongre** [reply]
I had gone through this tutorial. I have enabled 3D accel on the VM setting. However, it doesn't seem to be working as they mentioned it could be pretty slow.
In that case I can't run it on the VM.
